catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ValidateCommand.java

```diff
@@ -47,178 +55,104 @@
 @Service
 public class ValidateCommand implements Action {
 
-    @Argument(index = 0, name = "fileName", description = "The path to the file that you want to validate", required = true, multiValued = false)
+    @Option(name = "--path", aliases = "-p", description = "The path to the file to be validated")
```

Comment:
@coyotesqrl I was trying to allow the user to choose between providing a path or choosing to query the catalog. There may be a way using `@Argument` instead. I'll look into it